ovs_performance.py

```diff
@@ -816,7 +821,7 @@ def get_active_datapath_flows():
         cmd = 'sh -c "ovs-appctl dpctl/dump-flows system@ovs-system | wc -l"'
 
     result = dut_shell.dut_exec(cmd, die_on_error=True)
-    return int(result.stdout_output)
+    return int(result.stdout_output.decode("utf-8", "ignore"))
```

Comment:
I think we should fix this in the dut_ssh_shell.py module, to always return utf-8, so all changes in this file can be reverted.